client/src/components/ProgressBar/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Progress } from 'antd';
+import BackLink from '../BackLink';
+import { Container, Content } from './style';
+import { Body16B } from '../Typography';
```

Comment:
I think we should consistent with this syntax
```suggestion
import * as T from '../Typography';
```
and then you use it like this
`<T.Body16B color="blue">{currentLocation}</T.Body16B>`

Reply:
done